dmoj/cli.py

```diff
@@ -82,6 +83,11 @@ def cli_main():
     executors.load_executors()
     contrib.load_contrib_modules()
 
+    try:
+        readline.read_history_file(judgeenv.cli_history_file)
```

Comment:
This magically expands the ~ in path?

Reply:
Should be fixed now. Thanks for pointing that out.